MzTab_MTD

```diff
@@ -73,3 +73,40 @@ TABLESPACE pg_default;
 
 ALTER TABLE "MzTab"."Metadata"
     OWNER to postgres;
+    
+    
+-- Table: MzTab.Protein section
+
+-- DROP TABLE "MzTab"."Protein section";
+
+CREATE TABLE "MzTab"."Protein section"
+(
+    accession text COLLATE pg_catalog."default" NOT NULL,
+    description text COLLATE pg_catalog."default" NOT NULL,
+    taxid integer NOT NULL,
+    species text COLLATE pg_catalog."default" NOT NULL,
+    database text COLLATE pg_catalog."default" NOT NULL,
+    database_version text COLLATE pg_catalog."default" NOT NULL,
+    search_engine text[] COLLATE pg_catalog."default" NOT NULL,
+    "best_search_engine_score[1-n]" text COLLATE pg_catalog."default" NOT NULL,
+    "search_engine_score[1-n]_ms_run[1-n]" text[] COLLATE pg_catalog."default" NOT NULL,
```

Comment:
What do the "[1-n]" correspond to?  Are they needed?

Reply:
Index of variation (https://github.com/HUPO-PSI/mzTab/blob/master/examples/1_0-Proteomics-Release/SILAC_CQI.mzTab)
They are needed.